### PR TITLE
nodemailer

### PR DESCRIPTION
**Pull request by somu-code**
- Generated prisma schema for NextAuth
- Updated index page
- Created Card for /auth route
- Authentication with Github works now
- Now authenticated user can't access the /auth route
- Signin with email causing errors
- Signin via Github working
- Fixed naming
- Created Navbar and added to root layout
- Added example.env
- formated the code

### CODE REVIEW

**Comment by Praashh** [comment]
> * Generated prisma schema for NextAuth
> * Updated index page
> * Created Card for /auth route
> * Authentication with Github works now
> * Now authenticated user can't access the /auth route
> * Signin with email causing errors
> * Signin via Github working
> * Fixed naming
> * Created Navbar and added to root layout
> * Added example.env
> * formated the code

can you show, how the application is working via video/ss?

**Comment by somu-code** [comment]
> > * Generated prisma schema for NextAuth
> > * Updated index page
> > * Created Card for /auth route
> > * Authentication with Github works now
> > * Now authenticated user can't access the /auth route
> > * Signin with email causing errors
> > * Signin via Github working
> > * Fixed naming
> > * Created Navbar and added to root layout
> > * Added example.env
> > * formated the code
> 
> can you show, how the application is working via video/ss?

I will share it tomorrow. 
Tldr, nothing fancy just sign in with github.

**Comment by somu-code** [comment]
https://github.com/100XEnginners/CampusGhost/assets/60853878/c97a8930-0402-4346-9221-99039237379b

@9520prashant Here it is
UI and UX are not the best just making sure that functionality works for now.
